src/platform/web/Platform.js

```diff
@@ -174,10 +174,18 @@ export class Platform {
             const {body}= await this.request(this._configURL, {method: "GET", format: "json", cache: true}).response();
             this._config = body;
         }
-        this._notificationService = new NotificationService(
+        this.notificationService = new NotificationService(
```

Comment:
This was a bug from the previous PR which implemented reading config on app start.

Reply:
Noticed this too yesterday, have fixed it on master already :)